### PR TITLE
Add SCALE submissions.

### PR DESCRIPTION
**Pull request by castrojo**

### CODE REVIEW

**Comment by marcoceppi** [comment]
Let's make sure we name this "SCALE14x" or whatever scale it was. In the event of multiple years in the same conference a `-YEAR` or `-YEAR.MO` should be added

**Comment by castrojo** [comment]
I've left them all in one file for now so they don't clutter the directory and it makes it easier to see at a glance what has been submitted. Over time if we get a ton of submissions for one particular show we can break it down into a subdirectory.

File: topics_and_titles/SCALE.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Submissions to the Southern California Linux Expo
+
+List of submissions and text for SCALE:
+
+## 2015 - Real-world Big Data workloads in the cloud
```
Comment (marcoceppi) [inline topics_and_titles/SCALE.md]: I would prefer these were separate files, SCALEx14.md and SCALEx13.md etc

**Comment by marcoceppi** [comment]
I still think it's better to describe as different a file per conference

**Comment by marcoceppi** [comment]
:boat:

**Pull request merged**